pom.xml

```diff
@@ -202,6 +202,10 @@
                     <artifactId>commons-beanutils</artifactId>
                     <groupId>commons-beanutils</groupId>
                 </exclusion>
+                <exclusion>
+                    <artifactId>commons-collections</artifactId>
+                    <groupId>commons-collections</groupId>
```

Comment:
Is this correct? The `groupId` and `artifactId` are given same

Reply:
Yes, I could see it on the dependency tree. Also, ref: https://mvnrepository.com/artifact/commons-collections/commons-collections